<!-- test-analysis-auto -->
@claude 以下のテスト失敗を修正してください。

**失敗ジョブ**: Build
**Workflow Run**: https://github.com/sy-hash/mind-echo/actions/runs/22428496678

```
Build	Build for Testing	2026-02-26T04:58:26.3218830Z /Users/runner/work/mind-echo/mind-echo/MindEcho/MindEcho/Services/SummarizationService.swift:13:30: error: type 'LanguageModelSession' has no member 'isAvailable'
```

テストが通るようにコードを修正し、ビルドが通ることも確認してください。

--- a/MindEcho/MindEcho/Services/SummarizationService.swift
+++ b/MindEcho/MindEcho/Services/SummarizationService.swift
@@ -10,6 +10,9 @@ struct SummarizationService {
     }
 
     static var isAvailable: Bool {
-        LanguageModelSession.isAvailable
+        if case .available = SystemLanguageModel.default.availability {
+            return true
+        }
+        return false
     }
 }

--- a/MindEcho/MindEchoTests/SummarizationTests.swift
+++ b/MindEcho/MindEchoTests/SummarizationTests.swift
@@ -1,5 +1,6 @@
 import Testing
 import Foundation
+import Speech
 import MindEchoCore
 @testable import MindEcho
 

